src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+export const PeopleList: React.FC = () => {
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [error, setError] = React.useState<string | null>(null);
+  const [people, setPeople] = React.useState<Person[]>([]);
+
+  const { personSlug } = useParams();
+
+  React.useEffect(() => {
+    getPeople()
```

Comment:
Would be better to create a separate `async` function for it
It's best practice